src/Symfony/Component/Routing/Exception/MissingMandatoryParametersException.php

```diff
@@ -19,4 +19,36 @@
  */
 class MissingMandatoryParametersException extends \InvalidArgumentException implements ExceptionInterface
 {
+    private array $missingParameters = [];
+    private string $routeName = '';
+
+    public function __construct(/*array*/ $missingParameters, /*string*/ $routeName, /*int*/ $code = 0, \Throwable $previous = null)
```

Comment:
Prior to this change all constructor arguments were optional, now the first two are mandatory, this is a BC break if I'm not mistaking.

Reply:
Nice catch, thank you. It's fixed now !